modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinTcpIo.java

```diff
@@ -338,14 +339,19 @@ private HandshakeResult handshake(ClientListenerProtocolVersion ver) throws IOEx
 
             String err = reader.readString();
 
+            int status = reader.readInt();
```

Comment:
It is possible to get EOFException here, isn't ir?

Reply:
only when you will try to connect to server with version < 1.1.0